classes/Language.php

```diff
@@ -960,7 +960,17 @@ public static function downloadAndInstallLanguagePack($iso, $version = null, $pa
                 $files_list = $other_files;
             }
 
-            if (!$gz->extractList(AdminTranslationsController::filesListToPaths($files_list), _PS_TRANSLATIONS_DIR_.'../')) {
+            // don't know why, but it's needed & doesn't work otherwise
```

Comment:
@aleeks the translations_extract is needed and I believe it works, so which part it doesn't work?